src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule/ActivityHelper.cs

```diff
@@ -64,22 +64,35 @@ public static bool HasStarted(HttpContext context, out Activity? aspNetActivity)
         PropagationContext propagationContext = textMapPropagator.Extract(default, context.Request, HttpRequestHeaderValuesGetter);
 
         KeyValuePair<string, object?>[]? tags;
-        if (context.Request?.Unvalidated?.Path is string path)
-        {
-            tags = cachedTagsStorage ??= new KeyValuePair<string, object?>[1];
+        string? path = context.Request?.Unvalidated?.Path;
+        string? method = context.Request?.HttpMethod;
```

Comment:
I think if we need to set the normalized value as we do [here](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/7fcae4903133fb8d6e249963bceaa81689c5e326/src/OpenTelemetry.Instrumentation.AspNet/Implementation/HttpInListener.cs#L77)

If the method is normalized to `_OTHER` then should we also include [http.request.method_original](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/http.md), that part is not very clear in [spec](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-spans.md#http-server:~:text=implementations%20are%20possible.-,The%20following%20attributes%20can%20be%20important%20for%20making%20sampling%20decisions%20and%20SHOULD%20be%20provided%20at%20span%20creation%20time%20(if%20provided%20at%20all)%3A,-client.address)

Reply:
fixed [34e7fa3](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2023/commits/34e7fa38970f0b86c9e184540895b46e59534e8a)

Reply:
I think setting "HTTP" like the example you sent is the way to go. Keeping the convention.